code/__HELPERS/constants.dm

```diff
@@ -1,2 +1,2 @@
-var/global/const/TICKS_IN_DAY = 864000
-var/global/const/TICKS_IN_SECOND = 10
+var/const/TICKS_IN_DAY = 864000
```

Comment:
> Across different `world` objects.
What?

Are you implying it's possible to have a second `world` object *and* global variables being tied to it? Because that's not how global variables work in DM.